src/Common/Core/Impl/Install/ISupportedRVersionList.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+
+namespace Microsoft.Common.Core.Install {
+    public interface ISupportedRVersionList {
```

Comment:
This is closer to "supported version range"; "list" implies it's a collection, or at least enumerable.